### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files To Check 
`src/App.js`
`src/component/aside/Aside.jsx`
`src/component/carousel/Carousel.jsx`
`src/component/category/Category.jsx`
`src/context/product-context.js`
`src/index.js`
`src/pages/product/Product.jsx`

In this PR -  I integrated backend in the project and then added products, categories ,carousels data. Fetched the data using Axios and showing it on the view.
Thank you for reviewing my PR :)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import "./App.css";
-import { Navbar, Footer, Aside } from "./component";
-import { Home, Product } from "./pages/index";
```
Comment (hsnice16) [inline src/App.js:3]: Here, I think you can remove the `index` from the path like you are doing for the `components`.

**Review by hsnice16 — APPROVED** [review]
Good work, couldn't find much to suggest.

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

Liked that you have created a controller for the carousel as well

File: src/context/product-context.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import axios from "axios";
+
+const ProductContext = createContext();
+
+const ProductProvider = ({ children }) => {
+  const [productList, setProductList] = useState([]);
+  useEffect(async () => {
+    await axios
+      .get("/api/products")
+      .then((res) => setProductList(res.data.products));
```
Comment (arjun-computer-geek) [inline src/context/product-context.js]: you are mixing async await and .then with each other.

**Review by arjun-computer-geek — APPROVED** [review]
Change a suggested. Else looks good. Good folder structure and clean code. Well done.

File: src/App.js (hunk 1 above)
Comment (Developers-codz) [reply src/App.js:3]: Yeah, its mistake I guess thanks for pointing that out😅

File: src/context/product-context.js (hunk 2 above)
Comment (Developers-codz) [reply src/context/product-context.js]: Yes, correcting it thanks

**Pull request merged**